server/fishtest/templates/workers.mak

```diff
@@ -0,0 +1,70 @@
+<%inherit file="base.mak"/>
+
+<%!
+  import datetime
+  from fishtest.util import delta_date, diff_date
+%>
+
+<script>
+  document.title = 'Workers | Stockfish Testing';
+</script>
+<style>
```

Comment:
I advise against this added style as it breaks the style of the page.

![image](https://github.com/official-stockfish/fishtest/assets/63931154/18fb1ed2-27a0-4a20-9efd-6d355a2d7df8)

An improved style is something that can be worked on later. No need to worry too much about it right now imo.

Reply:
In workers.mak I added a bit of space after every div without using classes or anything, figuring this would be changed quickly anyway. However it seems this has a global effect.

Reply:
I have now protected it with a class. I called it .tmp since it should be changed. But with this PR I want to focus on the functionality.